packages/react/src/lib/accessibility/Behaviors/Toolbar/toolbarItemBehavior.ts

```diff
@@ -22,7 +22,10 @@ const toolbarItemBehavior: Accessibility<ToolbarItemBehaviorProps> = props => {
   behaviorData.keyActions.wrapper = {
     ...behaviorData.keyActions.wrapper,
     closeMenuAndFocusTrigger: {
```

Comment:
A bit worried that the name here implies close the menu and **focus the trigger**, but actually the combination `{ keyCode: keyboardKey.Tab, shiftKey: true }` is not focusing the trigger :\

Reply:
makes sense, @mnajdova. Created a separate action handler to handle that